web/src/pages/contact.tsx

```diff
@@ -1,20 +1,43 @@
 import { Box, Flex, Heading, Label, Link } from '@theme-ui/components';
 import React from 'react';
 import { SystemStyleObject } from 'theme-ui';
-import MapSvg from '../assets/map.svg';
 import { EmployeeList } from '../components/contact/exployee-list';
 import { Container } from '../components/container';
 import { Layout } from '../components/layouts';
 
+const empoyeeList = [
+  {
```

Comment:
I talk about this in #108 as well.

It is relevant here as well, IMHO. 

re https://github.com/bjerkio/website/pull/108#discussion_r600901491